crml/nft/src/lib.rs

```diff
@@ -207,6 +207,8 @@ decl_storage! {
 	trait Store for Module<T: Config> as Nft {
 		/// Map from collection to owner address
 		pub CollectionOwner get(fn collection_owner): map hasher(twox_64_concat) CollectionId => Option<T::AccountId>;
+		/// Map from collection to controller address
+		pub CollectionController get(fn collection_controller): map hasher(twox_64_concat) CollectionId => Option<T::AccountId>;
```

Comment:
would it be more useful to set controllers at the series level?

Reply:
conceptually a series should map to an erc721 contract which may have its own rules for minting otherwise need to make new collections each time